rcl_action/test/rcl_action/test_goal_handle.cpp

```diff
@@ -216,56 +217,58 @@ TEST_P(TestGoalHandleStateTransitionSequence, test_goal_handle_state_transitions
 
   // Walk through state transitions
   rcl_ret_t ret;
-  for (const EventStatePair & event_state : this->test_sequence) {
-    ret = rcl_action_update_goal_state(&this->goal_handle, event_state.first);
-    const rcl_action_goal_state_t & expected_state = event_state.second;
+  for (const EventStateActiveCancelableTuple & event_state : this->test_sequence) {
+    ret = rcl_action_update_goal_state(&this->goal_handle, std::get<0>(event_state));
```

Comment:
@jacobperron fyi: there's [`std::tie`](https://en.cppreference.com/w/cpp/utility/tuple/tie) to improve readability whenever one uses an `std::tuple`.

Reply:
Done 3a9be73